_posts/2017-09-05-life-at-codurance-robert-firek.md

```diff
@@ -0,0 +1,91 @@
+
+I actually joined Codurance and my first project was as a PHP developer, back before our main stacks matured around Java, .NET and Node.js. Within a few weeks however I was back to Java again (due to a change in the client project), and I haven’t written a single line of production code in PHP since. 
+
+It was probably the first time when I understood that I needed to be more than just the person who will be able to help them to shape their product by writing the code, but also by helping them improve their processes behind the software. It was a big challenge for someone without any experience in consultancy. Fortunately, all crafters in Codurance help you to develop the skills required to provide the best value for our customers. 
+
+After this project, I discovered a new passion :). A passion for DevOps. We were working a startup company that needed help to create their infrastructure in the cloud. This project was the most challenging project in my career. I had to learn a lot of new technologies and techniques to be able to provide the best value for our customer and to be the expert who can advise. At the same time, this startup hired one of the best people on the market, so together with my colleagues, we worked in the space where knowledge was flying in the air.
+
+<blockquote class="hero bq-text-lg"><p>"I discovered a new passion … A passion for DevOps"</p></blockquote>
+
+In the next project, I had to change my role completely. The software creation was our secondary goal. Together with other crafters, we were asked to help people to introduce the good practices like TDD. At the same time we trained the group of former Cobol programmers in Java. We run the study group where we were discovering the standards and libraries hidden behind Java. It was great pleasure to see the progress they made and how much energy they put to be Java developers. Thanks to the openness of our customer for new practices, we were able to organise the boot camp for our study group. During the two weeks of the boot camp, people had a chance to take part in the simulation of the real project. We received a lot of positive reviews but from my perspective we achieve more important goal. We managed to build the curiosity and willingness to learn. 
+
+### And what are you currently doing?
+
+I’m currently working with another Software Crafter alongside our client's development team to help improve the legacy code. Once again, this project is completely different from the former once. This team is very experienced in the domain (which is not trivial). Our main goal is to change architecture in the way which will allow reduce the required time to extend and / or introduce features. The company is not based in the UK which also gives the possibility to discover different cultures and food from other regions of Europe ;).
```

Comment:
Maybe "... this project is completely different from the former ones." ?

Reply:
"Our main goal is to change the architecture in a way which will allow them to reduce the required time to extend and/or introduce features." ?

Reply:
@Gryff What do you think about it: "Our main goal is to reduce the time required to introduce new features and extend the existing ones by improving the architecture of the application."